app/templates/components/upload.html

```diff
@@ -0,0 +1,87 @@
+{% set error_classes = 'file-upload-input file-upload-input-with-error invalid' if error else 'file-upload-input' %}
```

Comment:
yep, this is https://github.com/AusDTO/dto-digitalmarketplace-frontend-toolkit/blob/master/toolkit/templates/forms/upload.html but we refactored it into components at some point